plugins/polio/api.py

```diff
@@ -241,6 +243,53 @@ def create_calendar_xlsx_sheet(self, request, **kwargs):
         response["Content-Disposition"] = "attachment; filename=%s" % filename + ".xlsx"
         return response
 
+    @action(methods=["GET"], detail=False, serializer_class=None)
+    def csv_campaign_scopes_export(self, request, **kwargs):
+        groups = Group.objects.filter(id__in=request.GET.get("group", None).split(","))
+        org_units_list = []
+        for group in groups:
+            org_units = OrgUnit.objects.filter(groups=group.id)
```

Comment:
if you pass via the round.id you can districtely use the Campaign.get_district_for_round method to get the OrgUnits

Reply:
Done and thank you